native/src/modules/layout/components/Header.js

```diff
@@ -56,50 +57,38 @@ const BoxShadow: StyledComponent<{}, ThemeType, *> = styled.View`
 `
 
 type PropsType = {|
-  navigation: NavigationStackProp<*>,
-  scene: NavigationStackScene,
-  scenes: Array<NavigationStackScene>,
+  ...StackHeaderProps,
   t: TFunction,
   theme: ThemeType,
   peeking: boolean,
   categoriesAvailable: boolean,
-  navigateToLanding: () => void,
   goToLanguageChange?: () => void,
-  routeCityModel?: CityModel
+  routeCityModel?: CityModel,
+  dispatch: Dispatch<StoreActionType>
```

Comment:
I think dispatch should not leak into standalone components. That way this is only usable with redux.

Reply:
Agreed. However, this is similar to the behavior it was before (navigation was always passed) and I would refactor this separately.